client/src/styles/global.css

```diff
@@ -0,0 +1,51 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --cyf-color: #ff1e1e;
+  --cyf-background: #ffffff;
+  --milestone-table__wrapper: #808080;
+  --user__container: #000000:
```

Comment:
I think these color names are too descriptive as they are describing one use, I would go back to what you had

Reply:
done